tests/integration/test_ourlogs.py

```diff
@@ -136,35 +136,25 @@ def test_ourlog_extraction_with_sentry_logs(
 
     start = datetime.now(timezone.utc)
 
-    payload_with_all_fields = {
-        "timestamp": start.timestamp(),
-        "trace_id": "5b8efff798038103d269b633813fc60c",
-        "span_id": "eee19b7ec3c1b174",
-        "level": "info",
-        "body": "Example log record",
-        "severity_number": 10,
-        "attributes": {
-            "boolean.attribute": {"bool_value": True},
-            "sentry.severity_text": {"string_value": "info"},
-        },
-    }
-    envelope_with_all_fields = envelope_with_sentry_logs(payload_with_all_fields)
-    relay.send_envelope(project_id, envelope_with_all_fields)
-
-    payload_with_missing_fields = {
-        "timestamp": start.timestamp(),
-        "trace_id": "5b8efff798038103d269b633813fc60c",
-        "level": "warn",
-        "body": "Example log record",
-    }
-    envelope_with_missing_fields = envelope_with_sentry_logs(
-        payload_with_missing_fields
+    envelope = envelope_with_sentry_logs(
+        {
+            "timestamp": start.timestamp(),
+            "trace_id": "5b8efff798038103d269b633813fc60c",
+            "span_id": "eee19b7ec3c1b174",
+            "level": "info",
+            "body": "Example log record",
+            "severity_number": 10,
+            "attributes": {
+                "boolean.attribute": {"bool_value": True},
+                "sentry.severity_text": {"string_value": "info"},
+            },
+        }
     )
-    relay.send_envelope(project_id, envelope_with_missing_fields)
+    relay.send_envelope(project_id, envelope)
 
     ourlogs = ourlogs_consumer.get_ourlogs()
-    assert len(ourlogs) == 2
-    expected_with_all_fields = {
+    assert len(ourlogs) == 1
```

Comment:
Can just `assert ourlogs = [expected]` instead of asserting length and first element.